crane/api/repository.py

```diff
@@ -27,3 +27,9 @@ def get_tags_for_repo(repo_id):
     """
     # Validation that the repo exists is taken care of by the decorator
     return get_data()['repos'][repo_id].tags_json
+
+
+@authorize_name
+def get_path_for_repo(repo_id):
```

Comment:
Add a dockblock to this function, describing what it does, what repo_id is (with :param: and :type: tags), and what it returns (with :return: and :rtype: tags).